docs/extend/modular-resources/key-concepts.md

```diff
@@ -145,4 +145,4 @@ During robot shutdown, the RDK handles modular resource instances similarly to b
 ### Modular resources as remotes
 
 [Remote](/manage/parts-and-remotes/) parts may load their own modules and provide modular resources, just as the main part can.
-This means that you can compose a robot of any number of parts running in different compute locations, each containing both built-in and custom resources.
+This means that you can compose a smart machine of any number of parts running in different compute locations, each containing both built-in and custom resources.
```

Comment:
Should definitely be “robot” here for as long as the app says robot

Reply:
```suggestion
This means that you can compose a robot of any number of parts running in different compute locations, each containing both built-in and custom resources.
```